src/rcore.c

```diff
@@ -1534,6 +1534,16 @@ void SetWindowSize(int width, int height)
 #endif
 }
 
+// Set window opacity, value opacity is between 0.0 and 1.0
+void SetWindowOpacity (float opacity)
+{
+#if defined(PLATFORM_DESKTOP)
+    if (opacity >= 1.0) opacity = 1.0;
```

Comment:
float values should end with `f` --> `1.0f`, `0.0f`

Reply:
Yup, okay